README_DEVELOPERS.md

```diff
@@ -43,3 +43,17 @@ npm run test -- -- -t react
 ## Replace etalon
 
 To replace etalon image just remove it from `packages/devextreme-cli/testing/__tests__/__image_snapshots__` folder and run tests again
+
+## Run Updating Templates for Existing Applications
+
```

Comment:
I also suggest to describe the process of updating templates in more details. What applications will the script use to update templates? What is their location? How to initialize them(the command won't work if `sandbox` folder is empty)?